wordpress/wp-content/plugins/sil-dictionary-webonary/include/browseview_func.php

```diff
@@ -609,7 +618,8 @@ function reversalindex($display, $chosenLetter, $langcode, $reversalnr = "")
 		wp_register_style('reversal_stylesheet', $upload_dir['baseurl'] . '/' . $reversalCSSFile . '?time=' . date("U"));
 		wp_enqueue_style( 'reversal_stylesheet');
 
-		$arrReversals = getReversalEntries($chosenLetter, $pagenr, $langcode, $displayXHTML, $reversalnr);
+		$arrReversals = getReversalEntries($chosenLetter, $pagenr, $langcode, $displayXHTML, $reversalnr, $postsPerPage);
+		$totalEntries = $_GET['totalEntries'] ?? $wpdb->get_var('SELECT FOUND_ROWS()');
```

Comment:
This logic to get totalEntries from SELECT FOUND_ROWS() was mistakenly removed in one of the earlier code refactoring, which then broke paging in browse views.